src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -65,6 +67,10 @@ module.exports = Model.extend({
         this.fetch();
       }
     }, this);
+
+    this.on('change:localTimezone', function () {
```

Comment:
Yep. That's a prototype function and we were having bad problems. This way, it works. (But I don't know the real reason)